recipes/gulpio/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "gulpio" %}
+{% set version = "533.63" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cdc8c084bff58bdac25e6050f092ed395d883a9621844ffac0d6d6f394f32a6e
+
+build:
+  number: 0
+  skip: True  # [win] 
```

Comment:
Add `noarch: python` as this does not seem to require compiling per Python version.

Reply:
I wasn't able due to do this due to the selectors (see the conda linter report)

> `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

Reply:
What selectors do you need? I only see `skip: True. # [win]`, if it's pure python then why would you need to skip it on Windows?

Reply:
Because it uses the `sh` library which isn't supported on windows https://amoffat.github.io/sh/sections/faq.html#will-windows-be-supported

Reply:
This should be ok as long as the package is installable on Linux. My typical workflow is:

* Leave package without `noarch` and `skip: True  # [win]` in staged-recipes
* Add a comment why we do this
* When the feedstock was created, convert to `noarch: python` and remove the selector

The package will then be built on Linux with the sh dependency. When a windows user tries to install the package, conda will report correctly to the user that this is not possible as `sh` is not available on Windows.

Reply:
:+1: so once merged I should add the `noarch: python` back in?

Reply:
Yes.

Reply:
You can do it as noarch now. As long as circle passes, we can safely ignore windows failures.